mlibrary/search
# Custom `Tabs`, `Tab`, and `TabPanel` components.

# Overview
The original `Tabs` component used two dependencies: `react-tabs` and `@emotion/styled`. To replace those dependencies, `Tabs`, `Tab`, and `TabPanel` components were built from scratch. This now allows us to have full control.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Check out Advanced Search and Citations under full record views. Click around and see if anything is broken.
